src/fides/api/ops/schemas/messaging/messaging.py

```diff
@@ -132,47 +153,76 @@ class Config:
         extra = Extra.forbid
 
 
-class EmailConfigRequest(BaseModel):
-    """Email Config Request Schema"""
+class MessagingServiceSecretsTwilioSMS(BaseModel):
+    """The secrets required to connect to twilio SMS."""
+
+    twilio_account_sid: str
+    twilio_auth_token: str
+    twilio_messaging_service_sid: Optional[str]
+    twilio_sender_phone_number: Optional[
+        str
+    ]  # Either the twilio_messaging_service_id *OR* the twilio_sender_phone_number should be supplied.
```

Comment:
We should probably add a validator to make sure one or the other is not `None`. Or if one or the other is required could we make this one field with a more generic name and not make it `Optional`?

Reply:
I'd rather have explicit fields for these 2 configs. So I'll add a validator